Java/HernanCano/API_REST/README.md

```diff
@@ -0,0 +1,7 @@
+# REST API
+
+![RestApi](../../docs/img/RestApi.png)
```

Comment:
Decide if you are going to call it REST API or API REST, you shouldn't call the link and the image REST API and the directory API_REST